docs/src/manual/quad_forms/genusherm.md

```diff
@@ -80,19 +80,46 @@ There are two ways of creating a local genus symbol for hermitian lattices:
 We will construct two examples for the rest of this section. Note that the prime
 chosen here is bad.
 
-```@repl 2
```

Comment:
In this file, each example block re-defines everything it needs. Thus, we don't need to add an identifier (here `2`) to force all of them to be executed in the same namespace